### PR TITLE
Some fixes for phpdocs

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Review by fballiano — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

File: app/code/core/Mage/Bundle/Helper/Catalog/Product/Configuration.php [hunk 1]
```diff
@@ -87,9 +87,7 @@ public function getBundleOptions(Mage_Catalog_Model_Product_Configuration_Item_I
         $optionsQuoteItemOption = $item->getOptionByCode('bundle_option_ids');
         $bundleOptionsIds = $optionsQuoteItemOption ? unserialize($optionsQuoteItemOption->getValue(), ['allowed_classes' => false]) : [];
         if ($bundleOptionsIds) {
-            /**
-            * @var Mage_Bundle_Model_Resource_Option_Collection
-            */
+            /** @var Mage_Bundle_Model_Resource_Option_Collection $bundleOptionsIds */
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Bundle/Helper/Catalog/Product/Configuration.php:90]: Thanks. Will check this later.

[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Bundle/Helper/Catalog/Product/Configuration.php:90]: Fixed in #2602

File: app/code/core/Mage/Catalog/Model/Product/Option/Type/Select.php [hunk 2]
```diff
@@ -60,8 +60,7 @@ public function validateUserValue($values)
     /**
      * Prepare option value for cart
      *
-     * @throws Mage_Core_Exception
-     * @return mixed Prepared option value
+     * @return array|int|string|null Prepared option value
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Catalog/Model/Product/Option/Type/Select.php:63]: Fixed in #2602

File: app/code/core/Mage/Paypal/Model/Payflowlink.php [hunk 3]
```diff
@@ -635,8 +634,7 @@ protected function _checkTransaction($transaction, $amount)
      * Check response from Payflow gateway.
      *
      * @deprecated since 1.6.2.0
-     * @return Mage_Sales_Model_Abstract in case of validation passed
-     * @throws Mage_Core_Exception in other cases
+     * @return null in case of validation passed
```
[the remark this answers is not in the document]
Comment (sreichel) [reply app/code/core/Mage/Paypal/Model/Payflowlink.php:637]: Fixed in #2602

**Pull request merged**